### PR TITLE
Add note on running ARM64 containers on M1 machines

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Add a note on running ARM64 containers on Apple M1 machines.

### CODE REVIEW

File: docker-for-mac/apple-m1.md [hunk 1]
```diff
@@ -23,11 +23,14 @@ Click the following link to download the Apple M1 tech preview build:
...
 - The tech preview build does not update automatically. You must manually install any future versions of Docker Desktop.
-- You must install Rosetta 2 as some binaries are still Darwin/AMD64. To install Rosetta 2 manually from the command line use this command:
+- You must install Rosetta 2 as some binaries are still Darwin/AMD64. To install Rosetta 2 manually from the command line, use this command:
+
     ```
     softwareupdate --install-rosetta
     ```
+
 - The HTTP proxy is not enabled.
+- Attempts to run Intel-based containers on Apple M1 machines can crash as QEMU sometimes fails to run the container. Therefore, we recommend that you run ARM64 containers on M1 machines as these containers are also faster when compared to Intel-based containers.
```
Comment (stephen-turner) [inline docker-for-mac/apple-m1.md:33]: This and the next line are a bit contradictory. Is there a way we can combine them, or at least acknowledge the tension (eg start whichever we put second with "However, ...")?

Comment (stephen-turner) [reply docker-for-mac/apple-m1.md:33]: The second sentence is a bit unwieldy. Suggest split into two: "... on M1 machines. These containers will also be faster and use less memory than Intel-based containers."

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: docker-for-mac/apple-m1.md [hunk 2]
```diff
@@ -30,10 +30,13 @@ The tech preview of Docker Desktop for Apple M1 currently has the following limi
...
+- Not all images are available for ARM64 architecture. You can add `--platform linux/amd64` to run an Intel image under emulation. In particular, the [mysql](https://hub.docker.com/_/mysql?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image is not available for ARM64. You can work around this issue by using a [mariadb](https://hub.docker.com/_/mariadb?tab=tags&page=1&ordering=last_updated){: target="blank" rel="noopener" class=“”} image.
+
+   However, attempts to run Intel-based containers on Apple M1 machines can crash as QEMU sometimes fails to run the container. Therefore, we recommend that you run ARM64 containers on M1 machines. These containers are also faster and use less memory than Intel-based containers.
+
+- Some VPN clients can prevent the VM running Docker from communicating with the host, preventing Docker Desktop starting correctly.
```
Comment (stephen-turner) [inline docker-for-mac/apple-m1.md:38]: I suggest adding a link to https://github.com/docker/for-mac/issues/5208.

**Review by stephen-turner — DISMISSED** [review]
Looks good now. I've made one suggestion, but I'm happy to Approve with or without it.

File: docker-for-mac/apple-m1.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-mac/apple-m1.md]: Thanks. Added the reference.

**Review by stephen-turner — APPROVED** [review]

**Pull request merged**